### DISCUSSION TITLE
buddhism/2024/bujo/2024/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#128] [General]
# buddhism/2024/bujo/2024/

https://shanenull.com/buddhism/2024/bujo/2024/

**Comment by shane0** [comment]
https://github.com/shane0/buddhism/issues/97

**Comment by shane0** [comment]
# vows

- https://plumvillage.org/library/sutras/the-ten-great-aspirations-of-samantabhadra-bodhisattva
